glean-core/android/src/main/java/mozilla/telemetry/glean/private/TimingDistributionMetricType.kt

```diff
@@ -76,7 +76,7 @@ class TimingDistributionMetricType internal constructor(
             return null
         }
 
-        // The Rust code for [stopAndAccumulate] runs async and we need to use the same clock for start and stop.
+        // The Rust code for [start] runs async and we need to use the same clock for start and stop.
```

Comment:
Woops, good catch

Reply:
ooooh wait, the comment was correct, just not super clear.

`start` _does_ run sync (there's no dispatcher in use).
But `stop` runs async (we don't care about its return value) and THAT's why we need to use `getElapsedTimeNanos` in both because we need to use the same clock source.
I'm all for extending the comment, but the original was already more correct than after your change now.

Reply:
Ok -- I'll update this to be *both* correct *and* clear.